pymc3/distributions/continuous.py

```diff
@@ -1653,6 +1654,58 @@ def logcdf(self, value):
             tt.switch(tt.gt(y, 1), tt.log1p(-0.5 * tt.exp(-y)), tt.log(1 - 0.5 * tt.exp(-y))),
         )
 
+class AsymmetricLaplace(Continuous):
+    """
+    Assymetric Laplace Distribution
+    See also: https://en.wikipedia.org/wiki/Asymmetric_Laplace_distribution
+    """
```

Comment:
You might want to add more details in the docstrings, including the distribution parameters and respective domains, as is done for other distributions.

Reply:
okay , Thanks
I will add the docstrings.

Reply:
Hey, 
I feel like changing the parameter name of `AsymmetriLaplace` to its standard value. otherwise it will create problem while writing the `math` part of docstring.
can I change the parameter name?
and also, since `laplace_asymmetric` is still not available in `scipy.stats` distribution, should I import it while writing the `plot` part of docstring ? It will save out future work.

Reply:
I think you can skip the plot part of the docstring

Reply:
I don't know about the parameter thing. What is the problem it creates with the docstring?

Reply:
basically, while writing the math fromula in latex as it is there in other classes docstrings, the standard name like `kappa` instead of `symmetry` and `b` instead of `scale` would be appropriate.

Reply:
I don't have a strong opinion on that. I guess one argument in favor of using `b` is that the current `Laplace` distribution in Pymc3 uses `b` as well. This is kind of a running issue with PyMC, as in the `sd` vs `sigma` PR that is also open right now. Maybe someone else has a stronger opinion?

Reply:
okay.
Then, I am going with `b` and `kappa` instead of `symmetry` and `scale`.
If there is any objection, please do let me know.

Reply:
Yeah, we're using greek letters (for better or worse) in all other distributions so should keep that consistent.